bin/photos.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+'use strict'
+
+var easyimage = require('easyimage')
+var fs = require('fs')
+
+fs.readdir('./images/gallery', function (err, items) {
```

Comment:
**_So the current flow is drop the big images into the gallery folder, run the photos script to generate the thumbnails and then remove the large images again?_**
Correct.

**_It seems simpler to just add the image urls into the json file._**
Agree.
But the problem is in generating thumbs this way. Guess we would need to use the API for each platform where we're pulling the images from (one for twitter, another for Instagram, and so on...)
I also guess the module for generating thumbs doesn't work with URLs. 
You still need to provide it with a folder where to get the images from. 
But maybe I'm wrong, as I haven't fully analysed this.
One thing is for sure, the solution to automate this will require more thought and effort. 
But I can only dive into that on the next couple of days.
Maybe we can discuss this next Wednesday at the event, and you could give me your input on this.

**_I can imagine a situation where someone forgets to delete the original images and we end up with lots of large images in the repo._**
Agree. 
But thought we could go with a quick a simple gallery for now, and automate it latter.
Just guessing the risk you're mentioning wouldn't be that big after all. 

Cheers